lib/resolve-rc.js

```diff
@@ -8,16 +8,14 @@
  * @see https://github.com/babel/babel-loader/issues/62
  * @see http://git.io/vLEvu
  */
-var fs = require('fs');
-var path = require('path');
-var assign = require('object-assign');
-var exists = require('./helpers/exists')({});
-var read = require('./helpers/read')({});
+const path = require('path');
+const exists = require('./helpers/exists')({});
+const read = require('./helpers/read')({});
 
-var find = function find(start, rel) {
-  var file = path.join(start, rel);
-  var opts = {};
-  var up = '';
+const find = function find(start, rel) {
+  let file = path.join(start, rel);
+  //let opts = {};
```

Comment:
@danez - I couldn't think of nor find a reason for that `opts` object to exist. Can you think of a usage for it i'm not thinking of?